packages/jest-cli/src/cli/args.js

```diff
@@ -165,6 +165,11 @@ const options = {
       'other test output and user messages to stderr.',
     type: 'boolean',
   },
+  jsonOutputFile: {
+    description:
+      'Write the test results in JSON to your specified file.',
```

Comment:
Can we call this: 'Write test results to a file when the `--json` option is also specified.'